varcode/effects/effect_prediction_coding_in_frame.py

```diff
@@ -208,7 +208,7 @@ def predict_in_frame_coding_effect(
         reference_protein_length - aa_mutation_start_offset - 1)
 
     if mutant_codons_contain_stop and (
-            n_aa_alt < n_ref_amino_acids_after_mutated_site):
+            n_aa_alt <= n_ref_amino_acids_after_mutated_site):
```

Comment:
I was wondering if this condition is ok, but I was thinking it's not.

Suppose the coding sequence is "TTT|TGA", and we have a variant , TTTT>T at the start, meaning the coding sequence change is "TTT|TGA" > "TGA". Is this a PrematureStop or a Deletion?

This change seems like it would make it a PrematureStop?